AdminCore.Services/ContractService.cs

```diff
@@ -21,29 +21,28 @@ public ContractService(IDatabaseContext databaseContext, IMapper mapper) : base(
 
     public ContractDto GetContractById(int contractId)
     {
-      var contract = GetById(contractId);
-      return AddClientNameToContract(contract);
+      return _mapper.Map<ContractDto>(GetById(contractId));
     }
 
     public IList<ContractDto> GetContractByEmployeeId(int employeeId)
     {
-      var contract = DatabaseContext.ContractRepository.Get(x => x.EmployeeId == employeeId, null,
-                                              x => x.Team, x => x.Team.Client);
-      return ReturnContractDto(contract);
+      var contractList = DatabaseContext.ContractRepository.Get(x => x.EmployeeId == employeeId, null,
+                                              x => x.Team, x => x.Team.Project.Client);
+      return _mapper.Map<IList<ContractDto>>(contractList);
     }
 
     public IList<ContractDto> GetContractByTeamId(int teamId)
     {
-      var contract = DatabaseContext.ContractRepository.Get(x => x.TeamId == teamId, null,
-                                              x => x.Team, x => x.Team.Client);
-      return ReturnContractDto(contract);
+      var contractList = DatabaseContext.ContractRepository.Get(x => x.TeamId == teamId, null,
+                                              x => x.Team, x => x.Team.Project.Client);
+      return _mapper.Map<IList<ContractDto>>(contractList);
     }
 
     public IList<ContractDto> GetContractByEmployeeIdAndTeamId(int employeeId, int teamId)
     {
-      var contract = DatabaseContext.ContractRepository.Get(x => x.TeamId == teamId && x.EmployeeId == employeeId, null,
-                                              x => x.Team, x => x.Team.Client);
-      return ReturnContractDto(contract);
+      var contractList = DatabaseContext.ContractRepository.Get(x => x.TeamId == teamId && x.EmployeeId == employeeId, null,
```

Comment:
Unless the `x` is some kind of standard to specify parameters in lambda expressions, it should be `contract` in this case. That goes for all of these lambda expressions...

Reply:
Is refactoring existing code to have more meaningful variable names maybe outside the scope of this PR?

Reply:
If you are refactoring this function already I would say this change is acceptable, otherwise it will never get fixed.

Reply:
Refactored lambda function variable names.